Simple Webpage/Simple.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Color chooser</title>
+    <style>
+        #colorName {
+            border: none;
```

Comment:
@Jumeb i don't like this. First rename the file and use lower cases with underscores henceforth

Reply:
Same goes for folders without spaces. Also i asked you to use a color picker see one here http://jscolor.com/ . If you need help please ask here . I will help you out. Also i would love for the body background to be the key here not the background of the paragraphs
Looking forward to see those changes. And also drag folks here so the contribute